indico/web/client/js/jquery/extensions/global.js

```diff
@@ -247,19 +247,26 @@ $(document).ready(function() {
   showFormErrors();
 
   // Show form creation dialog if hash is present in the URL
-  var match = location.hash.match(/^#create-event:(lecture|meeting|conference)(?::(\d+))?$/);
+  let match = location.hash.match(/^#create-event:(lecture|meeting|conference)(?::(\d*)(?::([a-f0-9-]{36}))?)?$/);
   if (match) {
-    var eventType = match[1];
-    var categoryId = match[2];
-    var title = {
+    let eventType = match[1];
+    let categoryId = match[2];
+    let eventUUID = match[3];
+    let title = {
       lecture: $T.gettext('Create new lecture'),
       meeting: $T.gettext('Create new meeting'),
       conference: $T.gettext('Create new conference'),
     }[eventType];
-    var url =
-      match[2] !== undefined
-        ? build_url(Indico.Urls.EventCreation, {event_type: eventType, category_id: categoryId})
-        : build_url(Indico.Urls.EventCreation, {event_type: eventType});
+    let url;
+    if (match[2]) {
```

Comment:
Since you assign all those things to variables you can use them here as well instead of checking `match[x]` directly.